### PR TITLE
chore(docs): reorganize reference docs

### PR DESCRIPTION
**Pull request by rokroskar**
This moves around a few things in the docs to make it easier/more coherent with the changes introduced by https://github.com/SwissDataScienceCenter/renku/pull/2191. 

The docs look similar to before, but now with "Installation" and "getting started" in the sidebar along with "Reference".

image

The reason for this structure is so that we might in the future add "How-to guides" and similar docs that could be additionally reused in the main docs. 

The Reference page is where most of the content currently resides and looks like this:

image

### CODE REVIEW

**Review by vigsterkr — APPROVED** [review]
🚀

File: renku/cli/run.py [hunk 1]
```diff
@@ -181,8 +181,8 @@
 Exit codes
 ~~~~~~~~~~
 
-All Unix commands return a number between 0 and 255 which is called
-"exit code". In case other numbers are returned, they are treaded module 256
+All Unix commands return a number between 0 and 255 which is called an
+"exit code". In case other numbers are returned, they are treated modulo 256
```
Comment (Panaetius) [inline renku/cli/run.py:185]: :cry:

**Review by Panaetius — DISMISSED** [review]
Looks good to me, thank you!

File: renku/cli/run.py (hunk 1 above)
Comment (rokroskar) [reply renku/cli/run.py:185]: I honestly didn't know how to rephrase that...

File: renku/cli/workflow.py [hunk 2]
```diff
@@ -171,8 +171,8 @@
 
 Composing Plans into larger workflows
 *************************************
-.. image:: _static/asciicasts/compose_plan.delay.gif
-   :width: 600
+.. image:: ../_static/asciicasts/compose_plan.delay.gif
+   :width: 1000
```
Comment (vigsterkr) [inline renku/cli/workflow.py:175]: 💥

**Review by vigsterkr — APPROVED** [review]
:shipit:

**Review by Panaetius — DISMISSED** [review]
Thanks for looking into this!

File: renku/cli/workflow.py (hunk 2 above)
Comment (rokroskar) [reply renku/cli/workflow.py:175]: oops that was a mistake

**Comment by rokroskar** [comment]
sorry, fixed the typo - please re-approve 🙈

**Review by vigsterkr — APPROVED** [review]
🚢

**Review by Panaetius — APPROVED** [review]

**Pull request merged**